OWML.Common/MessageType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public enum MessageType
+    {
+        Error,
+        Warning,
+        Message,
+        Info,
```

Comment:
I really like the color of Info - but: what is the intended difference between Message and Info?

Reply:
I guess for things that should stand out to users, like mod updates, owml updates? not sure, but Rai mentioned it and I liked the idea... and I like the color 😋

Reply:
might be unnecessary though

Reply:
sold 👍